scripts/confluenceutils.py

```diff
@@ -11,16 +11,13 @@
 from jinja2 import Template
 import requests
 
-logging.basicConfig()
-logger = logging.getLogger("confluenceutils")
```

Comment:
The previous mechanism didn't work (info and below were not logged regardless of `--debug` so no data is actually lost in this transition. I'd argue that data is gained because the message we expected to be printed are now printed. 

However if you can point me at a simpler way to fix this, I'm game.